### ISSUE TITLE
Resources page API throttling

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#6856]
Inside each HTTP verb, a link should be given to where we can enable operational API throttling

Issues raised by Usability Testing candidates

**Comment by erangatl** [comment]
![image](https://user-images.githubusercontent.com/20179540/94636790-97155c00-02f3-11eb-948f-fff80c588b44.png)
If a user wants to enable Operation Level rate limiting, they have to scroll up. Many candidates struggled to find the radio buttons. Maybe we can give a link which will scroll up users up to the radio buttons.






![image](https://user-images.githubusercontent.com/20179540/94636749-70efbc00-02f3-11eb-89e3-3b003889e075.png)

**Comment by dushaniw** [comment]
Fixed with https://github.com/wso2/carbon-apimgt/pull/9295
